config/values.yaml

```diff
@@ -3,7 +3,9 @@
 controller_image: gcr.io/controller
 webhook_image: gcr.io/webhook
 build_init_image: gcr.io/build-init
+build_init_windows_image: gcr.io/cf-build-service-public/kpack/build-init-windows
```

Comment:
Is the expectation that this is built/updated out of band of ./hack/release.sh?

Reply:
yes, at least for now. Maybe in the future we could conceivably build the images using something other than pack that could run on linux and still build the windows images